docs/source/releases/v1.1.rst

```diff
@@ -73,6 +109,20 @@ Misc
 Migrations
 ~~~~~~~~~~
 
+Migrations will get picked up automatically for apps you haven't customised.
+If you have customised any app, please consult the
+`detailed instructions <topics/upgrading>`_ on how to handle migrations.
+
+.. warning::
+   This release contains a data migration for category slugs.  If you have
+   forked it, it is critical you run a copy of that migration when upgrading.
+
+Catalogue
+  * This app contains model changes.
+  * This app contains a data migration: ``0003_data_migration_slugs``.
+    It shouldn't matter whether you apply this migration before or after
+    the model changes.
```

Comment:
Because if you have forked the app, you need to manually declare the dependencies for the data migration. I hoped to have made that clear in my updates to ``topics/updating``. How can I phrase it better?